docs/users/fbc_workflow.md

```diff
@@ -0,0 +1,154 @@
+# FBC workflow
+
+The prerequisite for the File Based Catalog workflow is to convert the existing operator
+to FBC format. This can be done by following a [onboarding](./fbc_onboarding.md) documentation.
+
+After an existing operator is onboard or you introduce a brand new operator you can
+start the FBC workflow.
+
+## FBC templates
+File-based catalog templates serve as a simplified view of a catalog that can be updated
+by the user. The OPM currently supports 3 types of templates and it is up to the user which
+template the operator will be using.
+
+* Basic template
+* SemVer template
+* Composite template
+
+More information about each template can be found at [opm doc](https://olm.operatorframework.io/docs/reference/catalog-templates/).
+
+The recommended template from the maintainability point of view is `SemVer` or `Composite` with
+`SemVer`integration.
+
+## Generate catalogs using templates
+To generate a final catalog for an operator a user needs to execute different `opm`
+commands based on the template type. We as operator pipeline maintainers want
+to simplify this process and we prepared a `Makefile` with all pre-configured targets.
+
+To get the `Makefile` follow these steps
+
+```bash
+cd <operator-repo>/operator/<operator-name>
+wget https://raw.githubusercontent.com/redhat-openshift-ecosystem/operator-pipelines/main/fbc/Makefile
+```
+
+The right place for the Makefile is in the operator's root directory
+```
+.
+├── 0.0.1
+│   ├── manifests
+│   └── metadata
+├── catalogs.yaml
```

Comment:
Is the intent to have a `catalogs.yaml` for each operator?  The intention of this [spec](https://olm.operatorframework.io/docs/reference/catalog-templates/#specifications) is to allow the catalog owner to exert requirements on the operator author. So ideally this goes at the root of the whole repo, and expresses things like "we accept deliveries for 4.12...4.15 inclusive" or "for this particular catalog we'll take only semver submissions".

Reply:
I'm just now realizing that I never moved this file from example-operator-index to cool-catalog.  😢

Reply:
But [this](https://github.com/redhat-openshift-ecosystem/operator-pipelines/blob/main/docs/users/fbc_onboarding.md#generating-catalogs-from-templates) should refer to an http/https location to get general catalogs spec.

Reply:
Can you clarify?  Is this catalogs.yaml in the operator directory?  Common to the whole repo?